pkg/util/uploaderconfig/uploaderconfig.go

```diff
@@ -0,0 +1,54 @@
+package uploaderconfig
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+
+	velerov1api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+)
+
+const (
+	parallelFilesUpload = "ParallelFilesUpload"
+	writeSparseFiles    = "WriteSparseFiles"
+)
+
+func StoreBackupConfig(config *velerov1api.UploaderConfigForBackup) map[string]string {
+	data := make(map[string]string)
+	data[parallelFilesUpload] = strconv.Itoa(config.ParallelFilesUpload)
+	return data
+}
+
+func StoreRestoreConfig(config *velerov1api.UploaderConfigForRestore) map[string]string {
+	data := make(map[string]string)
```

Comment:
Will this cause inconsistency for different uploaders?
Can we be sure that the default value is always false for different uploaders?

Reply:
updated